test/functional/readme.md

```diff
@@ -8,137 +8,137 @@ Proposed functional tests are based upon Intern framework (https://theintern.io/
 ```sh
---testSuites=<test_suite_names_separated_by_comma>
-```
-- only for a specific stream or group of streams
-```sh
---stream=<stream or group name>
-```
-- to specify reporters
-```sh
---repoerters=<reporter_names_separated_by_comma>
+> java -jar selenium-server-standalone-3.4.0.jar -role node -nodeConfig Win10nodeConfig.json
 ```
 
-For example:
-- run all tests on chrome with pretty and junit reporters
-```sh
-node test/functional/runTests.js --os=windows --browsers=chrome --reporters=\"pretty,junit\"
-```
-- run only play and seek test suites
-```sh
-node test/functional/runTests.js --os=windows --browsers=chrome --testSuites=\"play,seek\"
-```
-- run all tests on stream "VOD (Static MPD) / SegmentBase, ondemand profile"
+When using driver for Edge version <=17 you need to adjust the "Dwebdriver.edge.driver" property in Selenium node configuration. 
+
 ```sh
-node test/functional/runTests.js --os=windows --browsers=chrome --stream=\"VOD (Static MPD) / SegmentBase, ondemand profile\"
+> java -jar selenium-server-standalone-3.4.0.jar -role node -nodeConfig Win10nodeConfig.json -Dwebdriver.edge.driver="msedgedriver.exe" 
 ```
 
+## Executing the tests
 
-## Running tests on Mac OSX
-#### Chrome
-Download latest chrome driver (chromedriver) from https://sites.google.com/a/chromium.org/chromedriver/downloads
-Make sure installation path is available in PATH variable.
-You can follow this article : http://www.kenst.com/2015/03/installing-chromedriver-on-mac-osx/
-
-#### Firefox
-Download last firefox driver (geckodriver) from https://github.com/mozilla/geckodriver/releases
-Make sure installation path is available in PATH variable.
-
-#### Launch the tests
-1. Start selenium hub. Open a new terminal and launch the command :
+To execute the tests you need to launch the intern runner. Open a new terminal at project root folder and launch the command:
 ```sh
-cd test/functional/selenium
-java -jar selenium-server-standalone-3.4.0.jar -role hub
+> node test/functional/runTests.js
 ```
 
-2. Start selenium node. Open a new terminal and launch the command :
+For the list fo available options run:
 ```sh
-cd test/functional/selenium
-java -jar selenium-server-standalone-3.4.0.jar -role node
+> node test/functional/runTests.js --help
 ```
-3. Start Intern runner. Open a new terminal and launch the command:
+
+The available options for running the tests are:
 ```sh
-cd test/functional
-./runTestsMac.sh
+runTests.js [options]
+
+Options:
+      --selenium    The selenium configuration preset name
+                                 [choices: "local", "remote"] [default: "local"]
+      --reporters   Reporters types (separated by ",", see intern.io
+                    documentation)                           [default: "pretty"]
+      --os          The OS platform on which tests must be executed (for test on
+                    local desktop, os is detected)
+                                [choices: "windows", "osx"] [default: "windows"]
```

Comment:
One minor correction here: browser.json uses the string "mac". Either change to osx or change this one to mac

Reply:
fixed